test/e2e/kind-conf.yaml

```diff
@@ -0,0 +1,9 @@
+# See https://kind.sigs.k8s.io/docs/user/configuration/
+# This allows starting a kind cluster with specific feature gates enabled
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+name: kind
+featureGates:
+  "StatefulSetMinReadySeconds": true
+networking:
+  ipFamily: ipv6
```

Comment:
We need to commit this if we want to run e2e tests that are feature gated but wasn't sure if this is the best directory?

It kind of makes sense here I think because it will serve as a point of reference for someone who wants to run the full e2e suite also?

Reply:
The other option would be under `scripts/` but no strong opinion from my side so lets keep it there.